.evergreen/config.yml

```diff
@@ -549,39 +552,6 @@ tasks:
       - func: run tests
         vars:
           UNIFIED: 1
-  - name: test-5.0-server
```

Comment:
Are all these removed lines with 5.0 tests and tls-support-4.4 intentional?

Reply:
Looks like the automatic merge conflict resolution may have messed with the generation script and resulting file